gf-dcc/portal
# Add devcontainer and improve docs

Adding devcontainer and docs to help @cconrad8, who would like to be able to contribute and update things as needed. Close issues #41 and  #45.

How to test/review:
1. On the `devcontainer` branch > "Create Codespace on devcontainer" from menu.
2. Go through https://github.com/gf-dcc/portal/tree/devcontainer#codespace and https://github.com/gf-dcc/portal/tree/devcontainer#updating-data.

Tagging @inodb for awareness and redundancy. 1) Go through same review steps if interested/have time. 2) Maybe more powerfully configure devcontainer by adding whatever extensions you guys actually use. I've been adding these across project repos because they help with reproducible development and makes it easier for more people on the team to contribute, especially if configuring their local environment is more painful than expected. So might be useful later for devs on the MSKCC team as well. 

More reading: https://www.infoq.com/articles/devcontainers/
